src/pages/Home.vue

```diff
@@ -55,20 +59,26 @@ import { useRouter } from 'vue-router';
 import { EXTERNAL_LINKS } from '@/constants/links';
 import TokenSearchInput from '@/components/inputs/TokenSearchInput.vue';
 import PoolsTable from '@/components/tables/PoolsTable/PoolsTable.vue';
+import FeaturedPools from '@/components/sections/FeaturedPools.vue';
 import usePools from '@/composables/pools/usePools';
 import useWeb3 from '@/services/web3/useWeb3';
 import usePoolFilters from '@/composables/pools/usePoolFilters';
+import { Network } from '@/constants/network';
 
 export default defineComponent({
   components: {
     TokenSearchInput,
-    PoolsTable
+    PoolsTable,
+    FeaturedPools
   },
 
   setup() {
     // COMPOSABLES
     const router = useRouter();
-    const { isWalletReady, isV1Supported } = useWeb3();
+    const { isWalletReady, isV1Supported, appNetworkConfig } = useWeb3();
+    const isEthereum = computed(() =>
```

Comment:
We have something similar with EIP1559 support and @TomAFrench came up with a nice feature based solution.
Essentially you add "supportsElementPools" to each config and that's when the feature is active.

Check `supportsEIP1559` for reference.

Reply:
ok thank

Reply:
resolved